.eslintrc.json

```diff
@@ -8,10 +8,10 @@
             "off",
             4
         ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
+        //"linebreak-style": [
+        //    "error",
+        //    "unix"
+        //],
```

Comment:
Comments aren't legal in JSON, so we should just remove this if it's a problem for Windows. 

@avandecreme I figure it's fine to allow different line endings... would you agree?